app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person_data in people:
+        name = person_data["name"]
+        age = person_data["age"]
+        person_list.append(Person(name, age))
```

Comment:
Consider using list comprehension to create the `person_list` as recommended in the checklist: 'When creating a list of instances from a collection, it is recommended to use list comprehension rather than traditional for loops.'